Test failure @mik-laj :

```
___________________ TestCliTasks.test_run_get_serialized_dag ___________________
  
  self = <tests.cli.commands.test_task_command.TestCliTasks object at 0x7f2bcf86d890>
  mock_local_job = <MagicMock name='LocalTaskJob' id='139826142040912'>
  mock_get_dag_by_deserialization = <MagicMock name='get_dag_by_deserialization' id='139826142042192'>
  
      @mock.patch("airflow.cli.commands.task_command.get_dag_by_deserialization")
      @mock.patch("airflow.cli.commands.task_command.LocalTaskJob")
      def test_run_get_serialized_dag(self, mock_local_job, mock_get_dag_by_deserialization):
          """
          Test using serialized dag for local task_run
          """
          task_id = self.dag.task_ids[0]
          args = [
              'tasks',
              'run',
              '--ignore-all-dependencies',
              '--local',
              self.dag_id,
              task_id,
              self.run_id,
          ]
  >       mock_get_dag_by_deserialization.return_value = SerializedDagModel.get(self.dag_id).dag
  E       AttributeError: 'NoneType' object has no attribute 'dag'
  
  tests/cli/commands/test_task_command.py:152: AttributeError
```

--- a/tests/cli/commands/test_task_command.py
+++ b/tests/cli/commands/test_task_command.py
@@ -73,7 +73,7 @@ class TestCliTasks:
         clear_db_runs()
 
         cls.dag = cls.dagbag.get_dag(cls.dag_id)
-        cls.dag.sync_to_db()
+        cls.dagbag.sync_to_db()
         cls.dag_run = cls.dag.create_dagrun(
             state=State.NONE, run_id=cls.run_id, run_type=DagRunType.MANUAL, execution_date=DEFAULT_DATE
         )

